src/service/resources-impl.js

```diff
@@ -686,9 +686,8 @@ export class Resources {
 
     if (this.documentReady_ && this.firstPassAfterDocumentReady_) {
       this.firstPassAfterDocumentReady_ = false;
-      this.viewer_.postDocumentReady(this.viewport_.getSize().width,
-        this.win.document.body./*OK*/scrollHeight);
-      this.updateScrollHeight_();
```

Comment:
Let's confirm with Eric. I believe these are no longer needed and I'd rather dump them ASAP. And I'm 99% sure that `updateScrollHeight_` is not needed.

Both of these were in play while we were pursuing the virtual viewport approach. That also reminds me: we no longer need `ViewportBindingVirtual_` either.

Let's close this PR. I'll confirm with Eric this and I'll send an overall PR that removes all of that old stuff.